frontend/src/routes/(root)/(logged)/scripts/run/[...hash]/+page.svelte

```diff
@@ -1,16 +1,17 @@
 <script lang="ts">
```

Comment:
you will want to carve out the changes as a component since this will be shared with the page for flow as well